### PR TITLE
Fix bugs related to version 8.0.0

### PR DESCRIPTION
**Pull request by alvarolaserna**
There are some bugs related to version 8.0.0 that are critical for the well functioning of TestUI.

To fix it we need version 8.1.1. 
Also location of chromedriver has changed.
There is also a new default path for appium server so we created `BASE_APPIUM_PATH` variable that defaults to /wd/hub/ so it works with appium version 1.X (2.X versions dont use that path and you will have to leave empty path)

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -135,7 +135,7 @@
         <dependency>
             <groupId>io.appium</groupId>
             <artifactId>java-client</artifactId>
-            <version>8.0.0</version>
```
Comment (aleksslitvinovs) [inline pom.xml:138]: Maybe keep this is a patch, i.e., `8.0.1`? 🤔

Comment (alvarolaserna) [reply pom.xml:138]: Yes, exactly my point

Comment (alvarolaserna) [reply pom.xml:138]: wait, you mean 2.0.1, the 8.0.0 change is the Appium version that has the fix

File: src/main/java/testUI/TestUIServer.java [hunk 2]
```diff
@@ -289,11 +289,14 @@ public static void stop(int driver) {
...
+                if (getDrivers().size() != 0)
+                    getDrivers().get(driver - 1).close();
                 sleep(500);
             }
-            getDrivers().get(driver - 1).quit();
-            removeDriver(driver - 1);
+            if (getDrivers().size() != 0) {
+                getDrivers().get(driver - 1).quit();
+                removeDriver(driver - 1);
+            }
```
Comment (aleksslitvinovs) [inline src/main/java/testUI/TestUIServer.java:299]: This looks sus. If iOS platform is used then driver can get deleted twice. Also, please wrap `if` statements in curly braces for readability

**Review by aleksslitvinovs — CHANGES_REQUESTED** [review]

File: src/main/java/testUI/TestUIServer.java (hunk 2 above)
Comment (alvarolaserna) [reply src/main/java/testUI/TestUIServer.java:299]: close and quit work differently, and for ios is better to run both, thats why it is ran like that

**Review by aleksslitvinovs — APPROVED** [review]

**Pull request merged**